src/components/search-bar/search-bar.tsx

```diff
@@ -61,17 +59,22 @@ export const SearchBar = withStyles(styles)(
             const { classes } = this.props;
             return <Paper className={classes.container}>
                 <form onSubmit={this.handleSubmit}>
-                    <input
+                    <Input
```

Comment:
Please check the component `SearchInput` maybe you could here re-use here

Reply:
They're similar, but search input has label and different style. Maybe later we could think about unification of those two, but for now I would leave it as it is